virttest/libvirt_xml/nwfilter_xml.py

```diff
@@ -384,3 +383,131 @@ def get_all_protocols(self, protocol=None):
                     protocols.append(new_one)
 
         return protocols
+
+
+class NwfilterBinding(base.LibvirtXMLBase):
+    """
+    Accessor methods for NwfilterXML class
+
+    Properties:
+        owner:
```

Comment:
suggest add simple NwfilterXML template example here

Reply:
```
<filterbinding>
  <owner>
    <name>rhel</name>
<uuid>c9543ff4-1703-40a9-abbf-7f3e0fcd66c0</uuid>
  </owner>
  <portdev name='vnet0'/>
<mac address='52:54:00:fa:81:87'/>
  <filterref filter='clean-traffic'>
<parameter name='MAC' value='52:54:00:fa:81:87'/>
  </filterref>
</filterbinding>
```

and i have separate it to a new file. please check

Reply:
i have separate the object to a file. And add the simple example